js/templates/modals/orderDetail/summaryTab/orderDetails.html

```diff
@@ -110,10 +110,23 @@ <h2 class="tx4 margRTn"><%= ob.polyT('orderDetail.summaryTab.orderDetails.headin
         </div>
       </div>
       <hr class="clrBr" />
-      <div class="gutterVTn">
-        <div class="txB">Memo</div>
-        <div><%= item.memo ? ob.parseEmojis(item.memo) : ob.polyT('orderDetail.summaryTab.notApplicable') %></div>
+      <div class="flexRow gutterH">
+        <div class="col6">
+          <div class="gutterVTn">
+            <div class="txB">Memo</div>
```

Comment:
Can we translate this? (I understand that was not an oversight on your part - I missed it)